tools/shared/src/main/scala/org/scalajs/core/tools/optimizer/OptimizerCore.scala

```diff
@@ -2165,6 +2165,37 @@ private[optimizer] abstract class OptimizerCore(
     }
   }
 
+  /** Translate literals to their ScalaJS String representation. */
+  private def foldToStringForString_+(tree: Tree)(implicit pos : Position): Tree = tree match {
+    case FloatLiteral(value)   =>
+      foldToStringForString_+(DoubleLiteral(value.toDouble))
+    case DoubleLiteral(value)  => 
```

Comment:
For cases whose body is on the next line, do not align the arrows. Also, they should have a blank line between them.